runtime/jcl/common/reflecthelp.c

```diff
@@ -428,19 +441,19 @@ getMethodParametersAsArray(JNIEnv *env, jobject jlrExecutable)
 j9object_t
 getMethodAnnotationData(struct J9VMThread *vmThread, struct J9Class *declaringClass, J9Method *ramMethod)
 {
-	return getAnnotationDataFromROMMethodHelper(vmThread, ramMethod, getMethodAnnotationsDataFromROMMethod);
+	return getAnnotationDataFromROMMethodHelper(vmThread, declaringClass, ramMethod, getMethodAnnotationsDataFromROMMethod);
 }
 
 j9object_t
 getMethodParametersAnnotationData(struct J9VMThread *vmThread, struct J9Class *declaringClass, J9Method *ramMethod)
 {
-	return getAnnotationDataFromROMMethodHelper(vmThread, ramMethod, getParameterAnnotationsDataFromROMMethod);
+	return getAnnotationDataFromROMMethodHelper(vmThread, declaringClass, ramMethod, getParameterAnnotationsDataFromROMMethod);
```

Comment:
Do you need to pass in declaringClass ? struct J9Method has a pointer to the constant pool.

Reply:
Then no I can use it from ramMethod